src/app/shared/myentries.service.ts

```diff
@@ -109,10 +109,10 @@ export abstract class MyEntriesService extends Base {
 
     groupEntries.forEach(groupEntry => {
       groupEntry.entries.sort((a, b) => {
-        if (a[path] < b[path]) {
+        if (a[path].toLowerCase() < b[path].toLowerCase()) {
```

Comment:
Is it safe to assume that the value of `path` is never null/undefined?

Reply:
`const path = type === EntryType.BioWorkflow || type === EntryType.Service ? 'full_workflow_path' : 'tool_path';` 
I think `path` will always be one of the two values. 
However, `a[path]` might be null/undefined